js/script.js

```diff
@@ -11,6 +11,12 @@ window.onload = function () {
     });
     createItems();
     createTiles();
+    // https://stackoverflow.com/questions/1586330/access-get-directly-from-javascript#1586333
+    window.$_GET = location.search.substr(1).split("&").reduce((o,i)=>(u=decodeURIComponent,[k,v]=i.split("="),o[u(k)]=v&&u(v),o),{});

```

Comment:
This line uses some ES6 features, which are not supported in IE11: http://kangax.github.io/compat-table/es6/

do we want to keep support for IE11?
 
Also, this line is very very hard to understand, can we please split this up into something readable?

Reply:
separated the code and replaced arrow function and return, did that fix it?